roles/keycloak/defaults/main.yml

```diff
@@ -0,0 +1,336 @@
+---
+# if the keycloak archive is from the "stage" branch, then you might enable the wayf plugin (keycloak-theme-vanilla)
+keycloak_archive_url: "https://github.com/keycloak/keycloak/releases/download/13.0.0/keycloak-13.0.0.tar.gz"
+
+##in format X.Y.Z
+## make sure the corresponding templates/X.Y.Z folder and vars/keycloak/X.Y.Z.yml exist
+keycloak_version: "13.0.0"
+
+keycloak_admin:
+  user: "<SET AN ADMIN USER>"
+  pass: "<SET AN ADMIN PASS>"
+
+
+keycloakdb:
+  name: "<KEYCLOAK DB NAME>"
+  user: "<DB USERNAME>"
+  pass: "<DB USER PASSWORD>"
+
+## set to true if this run should replace any existing installation (upgrade mode)
+is_upgrade: false
+
+# Email address for important account notifications. For example, Let’s Encrypt will automatically send
+# expiry notices to this email address when your certificate is coming up for renewal
+#letsencrypt_email: "bob.builder@example.org"
```

Comment:
Shouldn't this be part of the let's encrypt config?

Reply:
yes, indeed ! In rciam-deploy-inv was already there! 
Probably forgot it there when i was splitting them ;)